backend/src/server.ts

```diff
@@ -18,9 +18,27 @@ import { join } from 'path'
 import { URL } from 'url'
 import { promisify } from 'util'
 import { logError, logger } from './lib/logger'
+
 // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment,@typescript-eslint/no-var-requires
 const fastifyHttpProxy = require('fastify-http-proxy') // import isn't working for this lib.
 
+// const server = fastify()
+// const helmet = require('fastify-helmet')
+// fastify.use(require('hsts')())
+// server.register(helmet, { hsts: true })
+// const fastify = require('fastify')
+
+// fastify.use(require('hsts')())
+
+const fastify = require('fastify')()
```

Comment:
- fastify is imported in line 6 as `Fastify` (uppercase). You don't need this import.
- The `fastify` (lowercase) instance is created below around line 66-70.  So you need to move `fastify.register(...)` after line 70.